examples/for-tests/src/App.js

```diff
@@ -586,8 +586,8 @@ function getThirdPartyConfigs({ disableDefaultImplementation }) {
                 ThirdParty.Facebook.init(),
                 ThirdParty.Apple.init(),
                 {
-                    id: "custom",
-                    name: "Custom",
+                    id: "auth0",
+                    name: "Auth0",
```

Comment:
Do not remove custom.. add another one for auth0. Cause then other tests will fail.